### DISCUSSION TITLE
Library name adjustement

### DISCUSSION THREAD

**Discussion by jakoss** [s0nicyouth/kmapper#7] [Ideas]
Suggestion from my side - maybe the name kmapper isn't really that good for discoverability, because there already exists library with the same name doing (pretty much) the same thing: https://github.com/ProjectMapK/KMapper

This is extension of issue #3

**Comment by s0nicyouth** [comment]
@jakoss What would you suggest to rename it to?

**Reply by jakoss** [reply]
My first idea was Kapper but it's taken too: https://github.com/jchannon/Kapper

Maybe going with MapStruct relation this library could be called KapStruct? Although now i wrote it that doesn't seem too friendly :D

**Reply by s0nicyouth** [reply]
KapStruct sounds all right to me actually. So what do you think should be renamed? I think we can start from just renaming the repo and making adjustments to the Readme. Maven and package paths could so far stay the same.

**Reply by jakoss** [reply]
Well, there is no "kmapper" in maven url anyway which is a bit weird by itself, but it solves that problem :) But i think package paths might be worth changing now. It will get only harder the more popularity this library gets
